test/user_test.rb

```diff
@@ -0,0 +1,51 @@
+
+# pry(main)> sal.learn(joke_1)
+#
+# pry(main)> sal.learn(joke_2)
+#
+# pry(main)> sal.jokes
+# # => [#<Joke:0x00007fb71da169f0...>, #<Joke:0x00007fb71d8e0bd0...>]
+
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/joke'
+require './lib/user'
+
+class UserTest < Minitest::Test
+
+  def setup
+    @sal = User.new("Sal")
+    @ali = User.new("Ali")
+    @joke_1 = Joke.new(1, "Why did the strawberry cross the road?", "Because his mother was in a jam.")
+    @joke_2 = Joke.new(2, "How do you keep a lion from charging?", "Take away its credit cards.")
+  end
+
+  def test_it_exists
+    assert_instance_of User, @sal
+    assert_instance_of User, @ali
```

Comment:
this is fine, generally we only have to test this once; if it works for one case of `User.new` then it should work for all of them